inference-engine/thirdparty/movidius/XLink/pc/protocols/usb_boot.c

```diff
@@ -313,8 +313,10 @@ usbBootError_t usb_find_device_with_bcd(unsigned idx, char *input_addr,
             if (device) {
                 const char *dev_addr = gen_addr(dev, get_pid_by_name(input_addr));
                 if (!strcmp(dev_addr, input_addr)) {
+#if 0 // To avoid spam in Debug mode
```

Comment:
Is there a way to get the same behavior by using different log level?

Reply:
Unfortunately, no. This is the lowest level. It was ok before, when there was no mvLog

Reply:
Personally, I don't like that dead code. If we don't need these log even in debug mode, I'd remove it at all (what's the benefit anyway? That at some point we may want to look at it? It's so unlikely, that if we it is really so, we just will add additional log line and then would not commit it).

But I don't insist, resolving.

Reply:
This is important information when you debug this particular module. I agree that this is ugly, but it is better so far. It looks like we need an option to configure our log system for each module.